src/components/page/navbar.vue

```diff
@@ -69,7 +69,7 @@ export default {
 }
 
 #nav {
-    z-index: 1000;
+    z-index: 1000000;
```

Comment:
I think we can add a `z-index.css` like this:
```css
.low-z-index {
    z-index: 100;
}
.high-z-index {
    z-index: 1000;
}
.top-z-index {
    z-index: 100000;
}
```

Reply:
Bright idea